docker-init.sh

```diff
@@ -5,4 +5,5 @@ sed -i "s|https://demo.remark42.com|${REMARK_URL}|g" /srv/web/*.js
 # remove devtools attach helper. TODO: move to webpack loader
 sed -i "/REMOVE-START/,/REMOVE-END/d" /srv/web/iframe.html
 
+mkdir -p /srv/var
```

Comment:
this change effectively makes remark42 ephemeral in cases it no mounting provided. The side effects can be surprising for the user, i.e. restarted/recreated containers will lose all data.

Failing, in this case, feels like a better approach. I suggest adding an `if` check to the script and show some friendly message.

Reply:
Giving the commit history, I would argue that failing the start of the container without `/srv/var` was surprising for the user in the first place as it seems to be a semi-random side-effect of the change which broke it:)
I completely agree that it's better to document that behavior with a proper error message. I adjusted PR accordingly.